ckanext/datavicmain/views/datavic_main.py

```diff
@@ -134,11 +134,32 @@ def admin_report():
         return response
     return render('admin/admin_report.html', extra_vars={})
 
+def nominate_view(package_id,view_id):
+    try:
+        toolkit.get_action('nominate_resource_view')({}, {'package_id': package_id, 'view_id':view_id})
+        toolkit.h.flash_success('Successfully nominated view: %s' % view_id)
+    except Exception as e:
```

Comment:
1. Never catch `Exception`.
2. never `print` 
3. Don't call inside try/except anything that does not raise exception